README.hebrew.md

```diff
@@ -932,13 +932,14 @@ API, הודעה חדשה נרשמת לתור, וקריאה לכלי צפיה ב
 
 <br/><br/>
 
-## ![✔] 5.12. Strive to be stateless
+## ![✔] 5.12. תשאפו להיות נטולי-מצב
 
-**אמ;לק:** Store any type of _data_ (e.g. user sessions, cache, uploaded files) within external data stores. When the app holds data in-process this adds additional layer of maintenance complexity like routing users to the same instance and higher cost of restarting a process. To enforce and encourage a stateless approach, most modern runtime platforms allows 'reapp-ing' instances periodically
+**אמ;לק:** אחסנו כל סוג של מידע (לדוג׳ ביקור פעיל של משתמש, זכרון מטמון, קבצים שהועלו) באחסון נתונים חיצוני. שמירת נתונים בתוך תהליך התוכנה מוסיף עוד רובד של סיבוך בתחזוקה כמו ניתוב משתמשים לאותו מופע ועלות גבוהה יותר לאתחול מחדש של התהליך. על מנת לאכוף ולעודד תוכנה לנקוט בגישה נטולת-מצב, רוב פלטפורמות זמן הריצה מאפשרות לבצע 'reapp-ing' למופעים מעת לעת.

```

Comment:
@ShaiDemri Thanks for helping translate to Hebrew.

1. I guess that there is a mistake in the source about the `reapp-ing` because as far as I know this is not a valid word. @goldbergyoni What you think about the misspelling in the English version?
2. please add `:` after the `אחרת`.
3. please change the `קרא עוד` to `לקריאה נוספת`